ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -314,7 +314,8 @@ public enum TargetDotNetFrameworkVersion
         Version46 = 7,
         Version461 = 8,
         Version462 = 10,
-        VersionLatest = 10,
+        Version47 = 11,
+        VersionLatest = 11,
```

Comment:
@davkean agree the VersionLatest in enum isn't a good design. But when people install VS 2017 Update1 on clean machine, they will only get .net SDK 4.7 and Roslyn will break again as it looks for 4.6. We need fix that as well. 

My proposal is adding another ToolLocationHelper method or property to return latest version from reg key. When it is checked in, we might need change Roslyn to use the method/property instead of hardcoded Version46